docker/run_celery.bash

```diff
@@ -11,7 +11,8 @@ exec celery -A kobo worker --loglevel=info \
     --hostname=kpi_main_worker@%h \
     --logfile=${KPI_LOGS_DIR}/celery.log \
     --pidfile=/tmp/celery.pid \
-    --exclude-queues=sync_kobocat_xforms_queue \
+    --queues=kpi_queue \
+    --exclude-queues=kpi_low_priority_queue \
```

Comment:
I have to admit: I don't fully understand these options or why we went with `--exclude-queues` in the first place, if indeed we only wanted to run tasks from one queue. `celery worker --help` says this, which doesn't really answer my questions:
```
Queue Options:
  --purge, --discard
  -Q, --queues COMMA SEPARATED LIST
  -X, --exclude-queues COMMA SEPARATED LIST
  -I, --include COMMA SEPARATED LIST
```
Maybe this helps?
> A worker instance can consume from any number of queues. By default it will consume from all queues defined in the [task_queues](https://docs.celeryq.dev/en/stable/userguide/configuration.html#std-setting-task_queues) setting (that if not specified falls back to the default queue named celery).
>
> You can specify what queues to consume from at start-up, by giving a comma separated list of queues to the [-Q](https://docs.celeryq.dev/en/stable/reference/cli.html#cmdoption-celery-worker-Q) option: …

I'm just wondering if it would be clearer to say which queues we were consuming instead of which ones we weren't, especially if we add more than two queues in the future.

Reply:
I have to admit I was not sure myself. I kept `--exclude-queues` because of current syntax. But it would be more obvious to get rid of it.

Reply:
OK, let's stick with the tried-and-true for this quick change.